tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -277,7 +278,8 @@ def _zeros_slot(var, slot_name, op_name):
     self._get_or_make_slot = _get_or_make_slot
     self._get_or_make_slot_with_initializer = _get_or_make_slot_with_initializer
     self._zeros_slot = _zeros_slot
-  elif isinstance(self, optimizer_v2.OptimizerV2):
+  elif isinstance(self, optimizer_v2.OptimizerV2) or isinstance(
```

Comment:
Do you confirm this hack can be fitable for keras Optimizer?

Reply:
Yes, I confirmed it like Lifann said.

Reply:
After tf2.6, the keras has been moved out from TensorFlow.

It could import keras submodules as following:
```python
if tf.__version__ >= '2.6.0':
  from keras.optimizer_v2 import optimizer_v2
else:
  from tensorflow.python.keras.optimizer_v2 import optimizer_v2

if isinstance(self, optimizer_v2.OptimizerV2):
  ...
```

Reply:
And it would be better to use an independent commit to submit the python compat codes.

Reply:
> After tf2.6, the keras has been moved out from TensorFlow.
> 
> It could import keras submodules as following:
> 
> ```python
> if tf.__version__ >= '2.6.0':
>   from keras.optimizer_v2 import optimizer_v2
> else:
>   from tensorflow.python.keras.optimizer_v2 import optimizer_v2
> 
> if isinstance(self, optimizer_v2.OptimizerV2):
>   ...
> ```
I tried to refer to the keras containing optimizer_v2 as in the tf source code, but could not.